config/fcos/v1_6_exp/translate.go

```diff
@@ -376,3 +387,60 @@ func buildGrubConfig(gb Grub) string {
 	superUserCmd := fmt.Sprintf("set superusers=\"%s\"\n", strings.Join(allUsers, " "))
 	return "# Generated by Butane\n\n" + superUserCmd + strings.Join(cmds, "\n") + "\n"
 }
+
+func (c Config) handleSelinux(options common.TranslateOptions) (types.Config, translate.TranslationSet, report.Report) {
+	rendered := types.Config{}
+	ts := translate.NewTranslationSet("yaml", "json")
+	var r report.Report
+
+	for _, module := range c.Selinux.Module {
+		rendered = processModule(rendered, module, options, ts, r, path.New("yaml", "selinux", "module"))
```

Comment:
We should add the index to the path here so that e.g. module 2 is associated with the right output files and systemd units.

Reply:
Added the index path in the implementation and in the 'From Path'  in test files. :+1: